setup.py

```diff
@@ -104,8 +104,20 @@ def configuration(parent_package='', top_path=None):
 
     return config
 
+def is_Numpy_installed():
+    try:
+        import numpy
+    except ImportError:
+        return False
+    return True
 
 def setup_package():
+
+    build_requires = []
+    is_numpy_installed = is_Numpy_installed()
+    if is_numpy_installed is False:
+        build_requires = ['numpy>=1.6.2']
```

Comment:
We've had this discussion many times: do we actually want this? This will most likely result in people having numpy build on there system automatically, but without proper linear algebra compiled in it, which will give horrible run times.

I would much prefer a clean error message asking the user to install numpy, but no automatic build of numpy.